config/samples/ccruntime.yaml

```diff
@@ -7,5 +7,118 @@ spec:
   # Add fields here
   runtimeName: kata
   config:
+    cleanupNodeSelector: 
+      - matchLabels:
+        - name: "katacontainers.io/kata-runtime"
+          value: "cleanup"
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload:v0
+    payloadImage: quay.io/kata-containers/kata-deploy:stable
+    installDoneLabel:
+      katacontainers.io/kata-runtime: "true"
+    uninstallDoneLabel:
+      katacontainers.io/kata-runtime: "cleanup"
+    installerVolumeMounts:
+      - mountPath: /etc/crio/
+        name: crio-conf
+      - mountPath: /etc/containerd/
+        name: containerd-conf
+      - mountPath: /opt/kata/
+        name: kata-artifacts
+      - mountPath: /var/run/dbus
+        name: dbus
+      - mountPath: /run/systemd
+        name: systemd
+      - mountPath: /usr/local/bin/
+        name: local-bin
+    installerVolumes:
+      - hostPath:
+          path: /etc/crio/
+          type: ""
+        name: crio-conf
+      - hostPath:
+          path: /etc/containerd/
+          type: ""
+        name: containerd-conf
+      - hostPath:
+          path: /opt/kata/
+          type: DirectoryOrCreate
+        name: kata-artifacts
+      - hostPath:
+          path: /var/run/dbus
+          type: ""
+        name: dbus
+      - hostPath:
+          path: /run/systemd
+          type: ""
+        name: systemd
+      - hostPath:
+          path: /usr/local/bin/
+          type: ""
+        name: local-bin
+    installCmd: ["/opt/kata-artifacts/scripts/kata-deploy.sh", "install"]
+    uninstallCmd: ["/opt/kata-artifacts/scripts/kata-deploy.sh", "cleanup"]
+    cleanupCmd: ["/opt/kata-artifacts/scripts/kata-deploy.sh", "reset"]
```

Comment:
Just a note, I see you've group Install / uninstall / cleanup altogether, and I like it.  I like it very much that I wonder if there's some way to do the same with preInstall / postUninstall.

This doesn't have to be done right now, but that's something we can do before having a release out.
So, postUninstall / preInstall would become something like `Dependency`, and we could group in `installDependencyCmd`, and `uninstallDependencyCmd`, which would simplify our CRD.

@jensfr, @bpradipt, WDYT?

Reply:
@fidencio it's a good idea. I need to think about if it has other implications but I'll give it a show. As you said I think in a follow-on pull request. Thank you for the review!

Reply:
Addressing it in a follow-on PR is a good approach. May be creating an issue for this will be a good reminder.